UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPartSurgery.cs

```diff
@@ -28,7 +28,7 @@ public class SurgeryStep
 	{
 		public ItemTrait RequiredTrait;
 		public float Time; //ues tool
-		public int SuccessChance = 100;
+		public int FailChance;
```

Comment:
why

Reply:
Because unity inspectors idiot and defaults to 0 when making a new instance of it even though it says   = 100;